sdk/src/main/java/com/amplitude/experiment/ExperimentConfig.kt

```diff
@@ -179,6 +195,14 @@ class ExperimentConfig internal constructor(
             this.automaticExposureTracking = automaticExposureTracking
         }
 
+        fun pollOnStart(pollOnStart: Boolean) = apply {
+            this.pollOnStart = pollOnStart
+        }
+
+        fun fetchOnStart(fetchOnStart: Boolean?) = apply {
```

Comment:
The input value should not support null
```suggestion
        fun fetchOnStart(fetchOnStart: Boolean) = apply {
```